k8s/modules-ingress-dev.tf

```diff
@@ -1,3 +1,27 @@
+############CID Checker#######################################
+module "ingress-kong_cid-checker" {
+  count                            = local.is_dev_envs
+  source                           = "../modules/k8s_ingress"
+  get_global_configuration         = local.make_global_configuration
+  get_ingress_http_path            = "/"
+  get_ingress_backend_service_name = "cid-checker-frontend" // the "-service" string will be added automatically
+  get_ingress_backend_service_port = 1234
+  get_ingress_namespace            = kubernetes_namespace_v1.network.metadata[0].name
+  get_rule_host                    = "filecoin.tools"
+  type_lb_scheme                   = "external"
+}
+
+module "ingress-kong_cid-checker" {
```

Comment:
if you plan to deploy ingress for both dev and mainnet envs you can delete the count and second block.
if you need to keep both modules then resource names should be different

Reply:
fixed